client/lib/wp/browser.js

```diff
@@ -38,10 +38,8 @@ if ( config.isEnabled( 'support-user' ) ) {
 if ( 'development' === process.env.NODE_ENV ) {
 	require( './offline-library' ).makeOffline( wpcom );
 
-	// expose wpcom global var only in development
-	const wpcomPKG = require( 'wpcom/package' );
+	// expose wpcom global var in development mode
 	window.wpcom = wpcom;
-	window.wpcom.__version = wpcomPKG.version;
```

Comment:
Do you recall why this was necessary?

Reply:
I don't know. It was added in #4880 a _very_ long time ago, and seems quite unrelated to other changes in that PR.